LICENSING_POLICY.md

```diff
@@ -0,0 +1,64 @@
+# Ergo Licensing Policy
+
+In order to ensure an open source project is actually open and re-usable in a legal sense, 
+it’s critical that licensing information is clear. 
+Repositories in the `ergoplatform` and `ScorexFoundation` organizations should all follow this policy 
+for licensing their contents.
+
+Unless there is a particular reason to use a different licensing structure, please use 
+the following for new projects:
+
+1. All software code should be licensed under the MIT license. Canonical license text for 
```

Comment:
again, software is already under CC0 license mostly, not MIT

Reply:
Yes, except sigmastate, it is MIT. So should we change it to CC0?